Rethinking_2/Chp_07.ipynb

```diff
@@ -33,7 +33,7 @@
    "source": [
```

Comment:
<p>To get rid of the numerous <code>FutureWarning: Using from_pymc3 without the model will be deprecated in a future release</code>, can you replace <code>trace_m_sim = pm.sample()</code> with <code>trace_m_sim = pm.sample(return_inferencedata=True)</code> please?</p><p>That way, ArviZ doesn't have to convert the trace outside the model context in the <code>summary</code> call ;)</p><p>Nitpick but you can also replace <code>pm.summary</code> by <code>az.summary</code> as it's actually the function being called</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>ok</p>